tests/device_tests/bitcoin/test_fujicoin.py

```diff
@@ -45,5 +45,5 @@ def test_send_p2tr(client):
     # https://explorer.fujicoin.org/tx/a1c6a81f5e8023b17e6e3e51e2596d5b5e1d4914ea13c0c31cef90b3c3edee86
```

Comment:
Obviously not valid anymore. At least add a comment to explain.

Reply:
The same also applies to the BTC P2TR tests, a lot of which are on the blockchain, but will now have a different signature. Evidently `test_p2tr_with_proof` and `test_p2tr_fee_bump`, but also for example `test_send_mixed`, which is 37aa642d3cc0e0993c32e0213c92231841d3a45fcf0d03e7057ace75dd786146, and probably the others too.